### ISSUE TITLE
Found an unexpected Mach-O header code

### ISSUE THREAD

**Issue by fishandphil** [ParsePlatform/Parse-SDK-iOS-OSX#1110]
When I compile parse/bolt framework using the following: 

# To pull in extra dependencies (Bolts and OCMock)
git submodule update --init --recursive
# To install all the gems
bundle install
# Build & Package the Frameworks
rake package:frameworks

Everything work fine, however, when I try to upload to iTunes Connect, I get the following

**Found an unexpected Mach-O header code: 0x72613c21**

We are not using Carthage nor Pod on this project. If I revert to the previous version of Parse/Bolt, it work perfectly. 

Any idea?

**Comment by fauxpaw** [comment]
I believe I have the same issue. Adding the parse, bolts and parse UI frameworks gives me the same Mach-O header error when I try uploading to iTunes Connect. Still searching for a solution...

**Comment by flovilmart** [comment]
Have you seen that thread? http://stackoverflow.com/questions/33076819/found-an-unexpected-mach-o-header-code-0x72613c21-in-xcode-7

**Comment by fauxpaw** [comment]
I did look into those and so far nothing has fixed the issue. Some people have mentioned success with using an earlier version of the triggering libraries so I may fiddle around with them and see if it's a viable solution. Appreciate your looking into the issue flovilmart.

**Comment by flovilmart** [comment]
Are you using the dynamic or static flavor of the libraries? We just submit to the AppStore through cocoapods and no issue arised

**Comment by fauxpaw** [comment]
Found a solution that worked for me. Firstly, I was using Carthage to manage the dependencies for parse and bolts. (I was using dynamic flavor since I am developing via swift. My Mach-O build settings were set to executable) Carthage does not however support ParseUI so I had linked manually. This was all well and good for building and testing via device. To fix my Mach-O issues, I completely deleted Carthage and switched to Cocoa Pods. Pods supports both Parse and ParseUI. After managing the dependencies via cocoa pods, I was able to archive and am now in the midst of submitting to the store.
